src/components/utils/caseUtils.ts

```diff
@@ -168,7 +190,7 @@ export function useDownloadUtils() {
             messageTxt: errorMsg,
         });
 
-    const handleDownloadCases = async (
+    const handleExportCases = async (
```

Comment:
```suggestion
    const handleConvertCases = async (
```

to match the wording used in the UI
Otherwise it's difficult to understand what is different between downloading and exporting

Reply:
Done